### PR TITLE
[TRAFODION-2420] RMS enhancements

### PR DESCRIPTION
**Pull request by selvaganesang**
Introduced 2 new counters
HeapWM: - "SQL Heap WM" - Tracks the high water mark of the used memory
        including space buffer and bmo operator memory. It is enough to
        monitor this counter alone to track the memory utilization of the
        query.

scrIOTime: "Scr. IO Time" - Tracks the IO time taken to read/write to
        the scratch files at BMO operator level

Introduced a 'Single line Format' option to display BMO stats and SE stats
in a single line format to enable exporting the output to Excel easily.

GET STATISTICS FOR QID CURRENT PROGRESS, OPTIONS 'SL'

Removed and cleaned up the code further

Also fixed memory corruption issues seen with GET STATISTICS command.

### CODE REVIEW

File: core/sql/cli/sqlcli.h [hunk 1]
```diff
@@ -1169,15 +1170,14 @@ enum SQLSTATS_ITEM_ID {
...
-  SQLSTATS_OVEFLOW_PHASE_STARTTIME = 155,
-  SQLSTATS_OVERFLOW_PHASE = 156,
   SQLSTATS_SCRATCH_FILE_COUNT = 157,
   SQLSTATS_SCRATCH_BUFFER_BLOCK_SIZE = 158,
   SQLSTATS_SCRATCH_BUFFER_BLOCKS_READ = 159,
   SQLSTATS_SCRATCH_BUFFER_BLOCKS_WRITTEN = 160,
   SQLSTATS_SCRATCH_OVERFLOW_MODE = 161,
   SQLSTATS_SCRATCH_READ_COUNT = 162,
   SQLSTATS_SCRATCH_WRITE_COUNT = 163,
+  SQLSTATS_SCRATCH_IO_TIME = 155,
```
Comment (DaveBirdsall) [inline core/sql/cli/sqlcli.h]: Might be better to keep these in numeric order? I'm wondering what might happen if someone else added a new counter using 155, for example.

Comment (selvaganesang) [inline core/sql/cli/sqlcli.h]: I will move this entry to the deleted line below 1172 in a later check-in

**Comment by prashanth-vasudev** [comment]
+1 loog good to me.

**Comment by sureshsubbiah** [comment]
+1

**Pull request merged**